nana/modules/admins.py

```diff
@@ -153,194 +189,200 @@ async def pin_message(client, message):
 
 @app.on_message(filters.user(AdminSettings) & filters.command("mute", Command))
 async def mute_hammer(client, message):
-    if message.chat.type in ["group", "supergroup"]:
+    if message.chat.type in ['group', 'supergroup']:
+        chat_id = message.chat.id
```

Comment:
Codacy found an issue: [local variable 'chat_id' is assigned to but never used (F841)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330176)